bundled/umfpack/UMFPACK/Source/umf_grow_front.cc

```diff
@@ -112,7 +112,7 @@ GLOBAL Int UMF_grow_front
 	/* :: frontal matrix size int overflow :: */
 	/* the desired front size is bigger than the integer maximum */
 	/* compute a such that a*a*s < Int_MAX / sizeof (Entry) */
-	double a = 0.9 * sqrt ((Int_MAX / sizeof (Entry)) / s) ;
+	double a = 0.9 * sqrt (((double) Int_MAX / sizeof (Entry)) / s) ;
```

Comment:
```suggestion
	double a = 0.9 * sqrt ((double(Int_MAX) / sizeof (Entry)) / s) ;
```
Doesn't clang-tidy warn also about c-style casts?

Reply:
This is clang-10, not clang-tidy-10, but I assume that we disable c-style cast warnings in umfpack.

Reply:
This is also the style used in the rest of the file and I decided to rather follow it than to go with what we would do in the (main) library.